src/components/RepositoryGoals.js

```diff
@@ -45,6 +47,8 @@ function RepositoryGoals({user}) {
     dispatch({type: "UPDATE", payload: updatedRepos});
   };
 
+  const data = repository.data && JSON.parse(repository.data.text)
```

Comment:
Inconsistent syntax. A trailing semi-colon could be added to match other lines of code.